frontend/src/components/UpgradeDialog.tsx

```diff
@@ -0,0 +1,112 @@
+import { GemIcon, LifeBuoy, Mail, RefreshCw, Users, Zap } from "lucide-react";
+import { ReactNode } from "react";
+import { ExternalLinkButton } from "src/components/ui/button";
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "src/components/ui/dialog";
+import { useAlbyMe } from "src/hooks/useAlbyMe";
+import { useInfo } from "src/hooks/useInfo";
+
+interface UpgradeDialogProps {
+  children: ReactNode;
+}
+
+export const UpgradeDialog = ({ children }: UpgradeDialogProps) => {
+  const { data: albyMe } = useAlbyMe();
+  const { data: info } = useInfo();
+
+  if (albyMe?.subscription.buzz) {
+    return null;
+  }
+
+  return (
+    <Dialog>
+      <DialogTrigger>{children}</DialogTrigger>
+      <DialogContent className="max-w-md">
+        <DialogHeader>
+          <DialogTitle className="flex flex-row gap-2 items-center">
+            <GemIcon className="w-6 h-6 text-primary" />
+            Unlock Alby Pro
+          </DialogTitle>
+          <DialogDescription>
+            Take your Alby Hub experience to the next level
+          </DialogDescription>
+        </DialogHeader>
+        <div className="flex flex-col gap-6 my-4">
+          <div className="space-y-5">
+            <h3 className="font-medium text-lg">Pro Features Include:</h3>
+            <ul className="space-y-3">
+              <li className="flex items-center">
+                <Users className="w-5 h-5 mr-3 text-primary" />
+                <div>
+                  <span className="font-medium">Unlimited Sub-wallets</span>
+                  <p className="text-sm text-muted-foreground">
+                    Share with friends, family, coworkers
+                  </p>
+                </div>
+              </li>
+              {info?.backendType === "LDK" && (
```

Comment:
To also review in the onboarding / ConnectAlby page

Reply:
Since it was in there before, I'd postpone this one. I don't now how to handle an odd number of features in this list :sweat_smile: